islandora.module

```diff
@@ -2170,3 +2170,31 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $o = islandora_object_load($cmodel);
+      if (is_object($o)) {
+        $rows[] = array(l($o->label, "islandora/object/{$o->id}"));
+      }
+      else {
+        $rows[] = array(t("@cmodel  - (This content model is not in this Islandora repository.)", array('@cmodel' => $cmodel)));
+      }
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = array(
+    '#theme' => 'table',
+    '#header' => array($header),
+    '#rows' => $rows,
+    '#empty' => t('No Islandora content model associated with this object.'),
+  );
+  $rendered_table = drupal_render($table);
```

Comment:
I'll (re?) revert to format_plural (I had no idea that the translate() harvester could dig that far down,  Very cool).
 And yep, I should have said consistency instead of compatibility.  The collections table is functionally pretty similar in terms of exposing rdf relationship.  It looks pretty junky as an ordered list.